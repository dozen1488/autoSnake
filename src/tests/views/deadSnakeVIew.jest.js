/* eslint-env jest */

import React from "react";
import { mount, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";

import DeadSnakeRender from "../../application/views/deadSnakeView";

configure({ adapter: new Adapter() });

describe("Dead snake renders correctly", function() {
    it("equals with snapshot", function() {
        expect(
            toJson(mount(<DeadSnakeRender/>))
        ).toMatchSnapshot();
    });
});
