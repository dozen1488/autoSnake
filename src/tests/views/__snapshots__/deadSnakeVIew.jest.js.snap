// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dead snake renders correctly equals with snapshot 1`] = `
<renderDeadSnake>
  <div
    id="root-container"
  >
    <renderReadme>
      <div
        className="container"
      >
        <p
          className="readme-header"
        >
          Змейка с примитивным AI. Распознаёт окружение вокруг себя в радиусе 1 квадрата. При перезагрузке стреницы отсылает опыт на сервер. Сервер переобучает сеть и возвращает новую. Следовательно для обновления опыта змейки перезагрузите страницу. Пока что единственный способ сбросить опыт змейки до нуля - это: 
          <br />
          1) Остановить сервер 
          <br />
          2) Удалить фаил "/autosnake/server/images.json" 
          <br />
          3) Запустить сервер 
          <br />
        </p>
        <p
          className="readme-header"
        >
           &lt; Space &gt; - остановить змейку
        </p>
        <p
          className="readme-header"
        >
           &lt; Right Click &gt; - создать стену
        </p>
        <p
          className="readme-header"
        >
           &lt; Left Click &gt; - создать пищу
        </p>
        <p
          className="readme-header"
        >
          Запуск тестов: 
          <br />
          1) npm run karma - проверка ренедра квадрата и доски 
          <br />
          2) npm run jest - сравнение рендера примитивных компонентов со снимками
        </p>
      </div>
    </renderReadme>
    <p
      className="readme-header"
    >
      Snake has died. Refresh page please
    </p>
  </div>
</renderDeadSnake>
`;
